cli/dcoscli/main.py

```diff
@@ -89,43 +91,57 @@ def _main():
     subproc = Popen([executable,  command] + args['<args>'],
                     stderr=PIPE)
 
+    if log_level:
+        _config_log_level_environ(log_level, True)
+
+    if debug:
+        _config_debug_environ(debug, True)
+
     return analytics.wait_and_track(subproc)
 
 
-def _config_log_level_environ(log_level):
+def _config_log_level_environ(log_level, unset=False):
     """
     :param log_level: Log level to set
     :type log_level: str
+    :param unset: unset log setting
+    :type unset: bool
     :returns: True if the log level was configured correctly; False otherwise.
     :rtype: bool
     """
 
-    if log_level is None:
+    if unset:
         os.environ.pop(constants.DCOS_LOG_LEVEL_ENV, None)
         return True
 
-    log_level = log_level.lower()
+    log_level = ((log_level and log_level.lower()) or
+                 os.environ.get(constants.DCOS_LOG_LEVEL_ENV, None))
```

Comment:
The `None` argument is redundant.  `get` returns None by default if it can't find the key